src/OrbitGl/CaptureViewElement.cpp

```diff
@@ -124,7 +120,7 @@ std::vector<CaptureViewElement*> CaptureViewElement::GetChildrenVisibleInViewpor
   for (CaptureViewElement* child : GetNonHiddenChildren()) {
     float child_top_y = child->GetPos()[1];
     float child_bottom_y = child_top_y + child->GetHeight();
-    float screen_top_y = time_graph_->GetVerticalScrollingOffset();
+    float screen_top_y = 0;  // timeline_container_->GetVerticalScrollingOffset();
```

Comment:
Do you want to keep the comment?

Reply:
> Hi Florian, thanks for this! :)
> 
> I have a question. Did you test what's happening if we set TimeGraph to start at y = 20, instead of 0? I will need this soon because I would like to have the timeline in the top.

Right, I didn't think about that - this will again require a bit of refactoring, as currently TimeGraph assumes it fully occupies the `CaptureWindow`. We should discuss this, it also depends on how large the scrollbar will be in the future (i.e. if it fills the complete height of `CaptureWindow`, or if it stops below the timline.

Let's talk about the next steps before merging this.